plugins/openstack/keystone/check_keystone-api.sh

```diff
@@ -79,11 +88,11 @@ if [ -z "$TOKEN" ]; then
     echo "Unable to get a token"
     exit $STATE_CRITICAL
 else
-    if [ $TIME -gt 10 ]; then
-        echo "Get a token after 10 seconds, it's too long."
+    if [ $TIME -gt $TIMEOUT ]; then
+        echo "Got a token after $TIMEOUT seconds. That is too long."
```

Comment:
Mind rewording 'That is too long', bit more explanatory on what could potentially be wrong?